test/RevenuePool-test.js

```diff
@@ -1520,6 +1623,31 @@ contract("RevenuePool", (accounts) => {
         .estimateGas({ from: merchantSafe })
         .should.be.rejectedWith(Error, "Insufficient funds");
     });
+
+    it("rejects a claim when merchant safe is disabled and allows after re-enabled", async () => {
+      await merchantManager.disableSafe(merchantSafe);
+      expect(await merchantManager.getDisabledMerchantSafes()).to.have.members([
+        merchantSafe,
+      ]);
+
+      let amount = toTokenUnit(1);
+      let claimRevenue = revenuePool.contract.methods.claimRevenue(
+        daicpxdToken.address,
+        amount
+      );
+      // reverts are trigged via the gas estimation, so we'll never get far
```

Comment:
An alternative is using `.call`, like here: https://github.com/cardstack/card-pay-protocol/blob/main/test/PrepaidCardMarketV2-test.js#L337. Just FYI, no real need to fix this.